src/v/datalake/tests/arrow_writer_test.cc

```diff
@@ -86,7 +29,7 @@ test_int: int32
 test_long: int64
 test_float: float
 test_double: double
-test_decimal: decimal128(8, 16)
+test_decimal: decimal128(16, 8)
```

Comment:
This was a bug in the previous versions of the test code. Precision must be greater than scale, but it was reversed before. It didn't matter earlier because we weren't validating that we could translate the data to Parquet, but now that we are, this needs to be correct.